plugins/connector_api/connector_api.rb

```diff
@@ -76,4 +76,89 @@
     "password": keystore_password,
     "scope": "idsc:IDS_CONNECTOR_ATTRIBUTES_ALL"
   })
-end
\ No newline at end of file
+end
+
+endpoint '/api/v1/connectors/details/:client_id', ['GET'], public_endpoint: true do
+  begin
+    client_id = params['client_id']
+  rescue => e
+    halt 400
+  end
+
+  connector_details = load_connector_details(client_id, "id")
+  if connector_details.nil?
+    halt 404
+  end
+
+  halt 200, JSON.generate(connector_details)
+end
+
+endpoint '/api/v1/connectors/details', ['POST'], public_endpoint: true do
+  begin
+    json = JSON.parse request.body.read
+    client_name = json['client_name']
+    client_id = json['client_id']
+  rescue => e
+    halt 400
+  end
+  id_type = (client_id.nil? || client_id.empty?) ? "name" : "id"
+  used_id = (id_type == "name") ? client_name : client_id
+
+  connector_details = {}
+  if used_id.class == Array
+    used_id.each do |uid|
+      details = load_connector_details(uid, id_type)
+      if !details.nil?
+        connector_details[uid] = details
+      end
+    end
+  else
+    connector_details = load_connector_details(used_id, id_type)
+    if connector_details.nil?
+      halt 404
+    end
+  end
+
+  halt 200, JSON.generate(connector_details)
+end
+
+endpoint '/api/v1/connectors/remove', ['DELETE'], public_endpoint: true do
```

Comment:
what about designing the endpoint like this: DELETE /api/v1/connectors/{id} (path variable id = client_name and no request body)